config/initializers/bulkrax.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# Ensure Knapsack version gets loaded after Hyku's bulkrax.rb
+Rails.application.config.after_initialize do
+  # OVERRIDE Bulkrax v5.4.1
+  # Override default split regex to only split on unescaped comma and pipe chars
+  Bulkrax.instance_variable_set(:@multi_value_element_split_on, /\s*(?<!\\)[,|]\s*/)
```

Comment:
This definitely looks like we want a PR on Bulkrax for exposing a setter for this.

Reply:
What about leveraging `https://github.com/samvera/bulkrax/blob/64cf3143d1e9c2aef28a0ae4189c9e416b4c71d1/lib/bulkrax/entry_spec_helper.rb` for testing the split parsing?

Here's an example of testing CSV parsing:

https://github.com/scientist-softserv/adventist_knapsack/blob/7d0a5a91650873d8bd2e633dcb5871192fa9b1d5/spec/models/bulkrax/csv_entry_spec.rb#L24